cli/flox/Cargo.toml

```diff
@@ -1,6 +1,5 @@
 [package]
 name = "flox"
-version = "1.0.4"
```

Comment:
surprised that cargo will default to `0.0.0` if thre is no version field, i thought that was required.
anyway can we set the version to `0.0.0` explicitily like in the other crate(s)?

Reply:
`version` field is not mandatory. It is mandatory if you will publish the crate, which is not what we are doing.

Reply:
I'll set the version to `0.0.0` which is anyway the default version.